projects/fsastorefrontlib/src/cms-components/my-account/policy/policy.module.ts

```diff
@@ -72,9 +86,22 @@ const routes: Routes = [
       },
     }),
   ],
-  declarations: [PoliciesComponent, PolicyDetailsComponent],
-  exports: [PoliciesComponent, PolicyDetailsComponent],
-  providers: [PolicyService, ChangeRequestService],
-  entryComponents: [PoliciesComponent, PolicyDetailsComponent],
+  declarations: [
+    PoliciesComponent,
+    PolicyDetailsComponent,
+    PoliciesChartComponent,
+  ],
+  exports: [PoliciesComponent, PolicyDetailsComponent, PoliciesChartComponent],
+  providers: [
+    PolicyService,
+    ChangeRequestService,
+    PolicyChartDataService,
+    { provide: ChartConfig, useExisting: Config },
```

Comment:
I would move this to component itself, since there is no need for it to be available for other components in policy.module.
https://angular.io/guide/providers#limiting-provider-scope-with-components